src/Deprecated.jl

```diff
@@ -125,3 +125,18 @@ function updateVariableSolverData!(dfg::AbstractDFG,
   Base.depwarn("updateVariableSolverData! argument verbose is deprecated in favor of keyword argument `warn_if_absent`, see #643", :updateVariableSolverData!)
   updateVariableSolverData!(dfg, variablekey, vnd, solveKey, useCopy, fields; warn_if_absent = verbose)
 end
+
+
+##==============================================================================
+## Deprecated in v0.11 Remove in the v0.12 cycle
+##==============================================================================
+
+export AbstractRelativeFactor, AbstractRelativeFactorMinimize
+
+@warn("BREAKING CHANGE v0.12: deprecating AbstractRelativeFactor, use AbstractRelativeRoots instead")
```

Comment:
This looks like a good way to warn for deprecation. Perhaps it should be in __init so its called on using DFG?

Reply:
good idea thanks, 9cf4494